Automattic/wp-calypso
# Automattic for Agencies: Add Licenses page layout with search & filter

Resolves https://github.com/Automattic/jetpack-genesis/issues/240
Resolves https://github.com/Automattic/jetpack-genesis/issues/247
Resolves https://github.com/Automattic/jetpack-genesis/issues/248

<img width="1648" alt="Screenshot 2024-02-23 at 11 43 47 AM" src="https://github.com/Automattic/wp-calypso/assets/10586875/1b61cdbb-bcac-4410-b249-88fc9c885b9f">

## Proposed Changes

This PR adds:

- Layout for the Licenses page
- Search
- Filters

## Testing Instructions

NOTE: Please ignore any color difference for now. We will fix it later.

- Switch to the branch `git checkout add/a4a-licenses-layout-with-search-filter`.
- Start the server by running `yarn start-a8c-for-agencies`.
- Click the Licenses menu item > Verify that you can see the Licenses page as shown above.
- Verify that all the filters redirect you to appropriate links.
- Verify that searching for something appends the URL with the same text.
- Click the `Issue New License` button > Verify that you are redirected to `/marketplace`

## Pre-merge Checklist
